packages/zoe/src/contractFacet/zcfZygote.js

```diff
@@ -88,12 +89,13 @@ export const makeZCFZygote = async (
     instantiate: instantiateIssuerStorage,
   } = provideIssuerStorage(zcfBaggage);
 
-  /** @type {import('@agoric/swingset-vat').ShutdownWithFailure} */
+  /** @type {Panic} */
   const shutdownWithFailure = reason => {
     void E(zoeInstanceAdmin).failAllSeats(reason);
     seatManager.dropAllReferences();
     // https://github.com/Agoric/agoric-sdk/issues/3239
     powers.exitVatWithFailure(reason);
```

Comment:
Given that we panic below, I don't think any of these lines can have any effect.

```suggestion
```

Or better, just get rid of this function and have all uses just use `panic` directly.